### DISCUSSION TITLE
Automatic Edge Color

### DISCUSSION THREAD

**Discussion by grotesque** [freeplane/freeplane#1384] [General]
1. What does a column mean in Automatic Edge Color "for columns" in the right hand side menu format tab?
2. "Automatic Edge Color" is hidden from command search which makes it difficult to know about. I suggest, given the importance of this feature to me, that you must do something to make this feature known. [Others ](https://github.com/freeplane/freeplane/discussions/433)have requested this too.

**Comment by grotesque** [comment]
@dpolivaev

**Reply by dpolivaev** [reply]
I have pinned the discussion. Let us look if anyone responds.

**Comment by macmarrum** [comment]
```
Automatic edge color
    The color of branches and  nodes will be automatically generated following a defined scheme.
        for columns
            Nodes will have colors depending on horizontal distance from the root node.
            This option works only when all nodes of the mindmap have a fixed width.
                Nodes and summary nodes will always have alternating colors along the horizontal distance from the root node.
            Template file Bigmap.mm uses automatic edge color for columns.
```

![2024-05-25_12,28,40](https://github.com/freeplane/freeplane/assets/7197131/c7beffda-f524-4220-9357-8015d1cf145c)
